src/main/java/com/trueffect/logica/PersonValidationsDB.java

```diff
@@ -0,0 +1,73 @@
+package com.trueffect.logica;
+
+import com.trueffect.messages.Message;
+import com.trueffect.model.Person;
+import com.trueffect.response.ErrorResponse;
+import com.trueffect.sql.crud.PersonCrud;
+import com.trueffect.tools.CodeStatus;
+import com.trueffect.util.ErrorContainer;
+import java.sql.Connection;
+
+/**
+ * @author santiago.mamani
+ */
+public class PersonValidationsDB {
+
+    /*
+    * For dates duplidates
+     */
+    public static void veriryDataInDataBase(Connection connection, Person personNew) throws Exception {
+        ErrorContainer errorContainer = new ErrorContainer();
+        Person person = (Person) PersonCrud.getPersonByIdentifier(connection, personNew.getTypeIdentifier(), personNew.getIdentifier());
```

Comment:
Why the reason that you cast the response of PersonCrud.getPersonByIdentifier......